content/introduction-and-setup/next-steps.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Next Steps"
+date: 2022-12-15T09:16:07-06:00
+draft: false
+weight: 4
+originalAuthor: Sally Steuterman # to be set by page creator
+originalAuthorGitHub: gildedgardenia # to be set by page creator
+reviewer: # to be set by the page reviewer
+reviewerGitHub: # to be set by the page reviewer
+lastEditor: # update any time edits are made after review
+lastEditorGitHub: # update any time edits are made after review
+lastMod: # UPDATE ANY TIME CHANGES ARE MADE
+---
+
+## Next Steps
+
+Now that you have Visual Studio set up, you are ready to start programming in C#. The first step is to learn more about C# treats data and variables.
```

Comment:
Please rewrite the second sentence on line 17.

Reply:
I added more resources and info so this page can live its best life now.

Reply:
It would be nice to also include a brief sentence or two about the API documentation and resources Microsoft offers -> https://learn.microsoft.com/en-us/dotnet/csharp/

Reply:
Added that link and some info about why students might want to bookmark it.